cityofaustin/joplin
# 4169 import departments

# Description

When importing a page, we now create the appropriate group page permissions/department groups/department pages for it.


https://github.com/cityofaustin/techstack/issues/4169




# Types of changes



# How can this be tested?



 
* go here https://joplin-pr-4169-import-departme.herokuapp.com/
* try importing pages
* see department groups/pages get imported with them

# Checklist:
